src/components/notesCard/notescard.css

```diff
@@ -0,0 +1,32 @@
+.notes-card{
+    display: flex;
+    flex-direction: column;
+    height: max-content;
+    max-width: 15rem;
+    padding: 0.5rem;
+    padding-top: 1rem;
+    border-radius: 10px;
+    border: 0.1rem solid var(--primary);
+    color: var(--font-primary);
+    font-weight: 500;
+}
+.note-title{
+
+    margin-bottom: 0.5rem;
+}
+.note-action-icons{
+    margin-top: 1rem;
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    column-gap: 0.5rem;
+}
+.action-icon{
+    width: 2rem;
+    padding: 0.4rem 0.4rem;
+    fill: var(--font-secondary);
```

Comment:
Suggestion : its equivalent to padding : 0.4 rem , if possible use px values for values computing less than 10 px